### ISSUE TITLE
Gesture overrides broken in NVDA 2022.2.1

### ISSUE THREAD

**Issue by leonardder** [nvaccess/nvda#14065]
### Steps to reproduce:
Add the following gesture overrides for the brailliant driver. Providing the gestures.ini entries for convenience
```ini
[globalCommands.GlobalCommands]
braille_scrollForward = br(brailliantb):down
braille_scrollBack = br(brailliantb):up
braille_previousLine = br(brailliantb):left
braille_nextLine = br(brailliantb):right
```

### Actual behavior:
Overrides are ignored arbitrarily. For example, on several restarts of nvda:

| n | up | left | right | down |
| --- | --- | --- | --- | --- |
| 1 | braille_previousLine | braille_scrollBack | braille_scrollForward | braille_scrollForward |
| 2 | braille_previousLine | braille_previousLine | braille_nextLine | braille_nextLine| 
| 3 | braille_previousLine | braille_previousLine | braille_scrollForward | braille_scrollForward| 

### Expected behavior:
Gestures as provided in gestures.ini are always mapped

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2022.2.1 and recent alpha

#### Windows version:
Windows 11 22621.105

#### Name and version of other software in use when reproducing the issue:
n/a

#### Other information about your system:
Humanware Mantis connected via USB. Also reported by @dkager when using a Brailliant.

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, does not occur with 2022.2

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by leonardder** [comment]
Probably related to changing globalMapScripts.extend to globalMapScripts.update in scriptHandler.

**Comment by seanbudd** [comment]
This was closed by #14066 correct?
